nshlib/nsh.h

```diff
@@ -1204,6 +1204,10 @@ int cmd_pmconfig(FAR struct nsh_vtbl_s *vtbl, int argc, char **argv);
   int cmd_usleep(FAR struct nsh_vtbl_s *vtbl, int argc, char **argv);
 #endif
 
+#ifndef CONFIG_NSH_DISABLE_UPTIME
+  int cmd_uptime(FAR struct nsh_vtbl_s *vtbl, int argc, char **argv);
```

Comment:
```suggestion
  int cmd_uptime(FAR struct nsh_vtbl_s *vtbl, int argc, FAR char **argv);
```
but seems to be a common problem for this file

Reply:
Done.  I checked all the source files under nshlib and it looks like a lot of them are missing the FAR keyword, let's merge this PR first, and then I will send another PR to add the missing FAR keyword for all source files under nshlib.